tests/utils.go

```diff
@@ -1940,16 +1940,29 @@ func NewRandomFedoraVMIWithDmidecode() *v1.VirtualMachineInstance {
 }
 
 func GetGuestAgentUserData() string {
+	virtClient, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
+
+	dnsServerIP, err := getClusterDnsServiceIP(virtClient)
+	PanicOnError(err)
+
+	ipv6UserDataString := ""
+	if netutils.IsIPv6String(dnsServerIP) {
+		ipv6UserDataString = fmt.Sprintf(`sudo ip -6 addr add fd2e:f1fe:9490:a8ff::2/120 dev eth0
+                             sudo ip -6 route add default via fd2e:f1fe:9490:a8ff::1 src fd2e:f1fe:9490:a8ff::2
```

Comment:
I see that we're hard-coding the IP (also route) in different places in the tests (at least in the cloud init & in the expector). 

Furthermore, this IP has to match what is being assigned by the https://github.com/kubevirt/kubevirt/pull/3112/files#diff-d3ed8eb8cca020f37cab207c079f2627R563 algorithm. 

That's not good, and should be addressed either in this, or on a follow up PR.

Reply:
This PR doesn't include adding vmIpv6NetworkCIDR to podInterface schema.
It supports only the default one.

It would be addressed in a follow up pr, please refer to the PR message.